package.yaml

```diff
@@ -0,0 +1,37 @@
+name:                github-api
+version:             0.1.0.0
+github:              "githubuser/github-api"
+license:             BSD3
+author:              "Author name here"
+maintainer:          "example@example.com"
+copyright:           "2018 Author name here"
```

Comment:
This block is like 30% filled in.

Reply:
I use by-default .cabal file for that. Will change it there.